modules/imagick/manifests/init.pp

```diff
@@ -1,10 +1,18 @@
 class imagick (
-$path = "/vagrant/extensions/imagick"
+	$path = "/vagrant/extensions/imagick",
+	$php = $imagick_config[php]
 ) {
 
-  package { 'php5-imagick':
-    ensure  => latest,
-    require => Package['php5-fpm'],
-    notify  => Service['php5-fpm']
-  }
+	if versioncmp( "${$php}", '5.4') <= 0 {
+		$php_package = 'php5'
+	}
+	else {
+		$php_package = "$php"
+	}
+
+	package { "${$php_package}-imagick":
+		ensure  => latest,
+		require => Package["php${$php_package}-fpm"],
```

Comment:
Can you change this and the line below to be `Package["${$php_package}-fpm"]` and I think we should be all good. Thanks!